receiver/postgresqlreceiver/client.go

```diff
@@ -789,18 +790,16 @@ func indexKey(database, schema, table, index string) indexIdentifer {
 //go:embed templates/querySampleTemplate.tmpl
 var querySampleTemplate string
 
-func (c *postgreSQLClient) getQuerySamples(ctx context.Context, limit int64, logger *zap.Logger) ([]map[string]any, error) {
+func (c *postgreSQLClient) getQuerySamples(ctx context.Context, limit int64, newestQueryTimestamp float64, logger *zap.Logger) ([]map[string]any, float64, error) {
 	tmpl := template.Must(template.New("querySample").Option("missingkey=error").Parse(querySampleTemplate))
 	buf := bytes.Buffer{}
 
-	// TODO: Only get query after the oldest query we got from the previous sample query colelction.
-	// For instance, if from the last sample query we got queries executed between 8:00 ~ 8:15,
-	// in this query, we should only gather query after 8:15
 	if err := tmpl.Execute(&buf, map[string]any{
-		"limit": limit,
+		"limit":                limit,
+		"newestQueryTimeStamp": newestQueryTimestamp,
```

Comment:
nit: Would be nice to be consistent: either "TimeStamp" or "Timestamp" everywhere. Inconsistency increases the likelihood of introducing bugs in the future.

Reply:
updated

Reply:
Thanks!